api/end_device.proto

```diff
@@ -285,7 +285,7 @@ message MACSettings {
 
 // MACState represents the state of MAC layer of the device.
 // MACState is reset on each join for OTAA or ResetInd for ABP devices.
-// This is used internally by the Network Server and is read only.
+// This is used internally by the Network Server.
```

Comment:
Should we specify that it is partially read only?
This field is really low-level API and is generally meant to only be modified by Network Server itself.